src/AdminAcceptanceTests.Actions/Pages/UserAccountsDashboard.cs

```diff
@@ -44,7 +44,6 @@ public bool AddUserButtonDisplayed()
         public void ClickAddUserButton()
         {
             Driver.FindElement(Pages.UserAccountsDashboard.AddUser).Click();
-            Wait.Until(d => d.FindElements(Pages.UserAccountsDashboard.AddUser).Count == 0);
```

Comment:
not a fan of waiting for the add button to disappear?

Reply:
Doesn't need to wait, plus I prefer to wait for things to appear, rather than disappear...